docs/ref/models/fields.txt

```diff
@@ -319,6 +319,41 @@ characters that aren't allowed in Python variable names -- notably, the
 hyphen -- that's OK. Django quotes column and table names behind the
 scenes.
 
+
+``db_comment``
+--------------
+
+.. versionadded:: 4.0
+.. attribute:: Field.db_comment
+
+db_comment support SQL COMMENT.
+
+A string describing the Model field which will be saved as a SQL
+COMMENT on the SQL COLUMN
+
+Unlike `help_text`, db_comments are displayed in the database as
+SQL COMMENTs, which makes them useful for documenting fields
+for database administrators, developers, analysts, and others with
+direct database access, especially if you need to provide different
+explanations  for users or site admins (using `help_text`) vs. for DBAs
+(using `db_comment`). For example::
+
+    db_comment = "User karma points. This is field is denormalized,\
+                          so should not be updated alone. This comment is\
+                          managed by Django, and will be updated on every\
+                          database migration."
+
+db_comment has a character limit that differs for each database
+backend. PostgreSQL has a 1GB limit. MySQL has a 1024 character limit.
+
+.. warning::
+    Changes to database column comments made by database
+    users outside of Django will be overwritten by Django migrations
+    if Django finds a value for db_comment on a model field.
+    Communicating with your DBAs and issuing reminder notices in
+    the comments (as in the example above) may help avoid conflicts.
```

Comment:
I think I mentioned this before, but as it applies to more or less everything in Django, I think we don't need it.

Reply:
I agree. This can be removed.